### ISSUE TITLE
Error that causes bridge to constantly reset

### ISSUE THREAD

**Issue by jfox815** [dustindclark/homebridge-envisalink#76]
I just tried to install this plugin and it was causing the bridge it was on to crash and reset constantly. I’m running HOOBS 4.0.100. 

Has anyone else had this problem?

**Comment by dustindclark** [comment]
Can you try again with the latest version of the plugin?

**Comment by benchristian88** [comment]
I was having he same issue. Latest plugin version.  latest homebridge.

It initializes, generates all the uuid, starts the proxy, initializes the accessories,  shows the enter code to connect to homebridge, and then an error and restarts hombridge.

Figured out that it was because I already have my vera polling the evl4, and I just powered that down, rebooted the card, and then this plugin worked perfectly, and then the alarmed worked in home.

Need to test it all but looks good. 

Turns out the evl card can only have one device polling at a time.  once i realised that I am happy.  nice work.
